packages/odyssey-storybook/src/components/odyssey-mui/Status/Status.stories.tsx

```diff
@@ -25,8 +25,13 @@ const storybookMeta: Meta<StatusProps> = {
       control: "radio",
       options: ["default", "error", "info", "success", "warning"],
     },
+    indicator: {
+      control: "radio",
+      options: ["lamp", "pill"],
+    },
   },
   args: {
+    indicator: "lamp",
     label: "Warp drive in standby",
     severity: "default",
   },
```

Comment:
Previously, we didn't have two variants, so selecting the variant wasn't required. On the design side, we do want people to default to `lamp` and only use `pill` when they need to differentiate.

Reply:
I'm okay calling this `variant` instead of `indicator`. The naming here was forward-looking, as there are other variants that _only_ change the indicator (icon and descender) planned for the future.